Pytest fails on master branch

There is a `pytest` failure in the master branch, and I'm not sure how to fix it. I concluded it occur on master with pandas=1 and pandas<1. It feels bad to make another release before this is fixed or understood.

## Duplicated issue
- https://github.com/quantopian/qgrid/issues/274

## pytest error logs

```
============================================= FAILURES ==============================================
_____________________________________ test_period_object_column _____________________________________

    def test_period_object_column():
        range_index = pd.period_range(start="2000", periods=10, freq="B")
        df = pd.DataFrame({"a": 5, "b": range_index}, index=range_index)
>       view = QgridWidget(df=df)

qgrid/tests/test_grid.py:240: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
qgrid/grid.py:627: in __init__
    self._update_df()
qgrid/grid.py:820: in _update_df
    self._update_table(update_columns=True, fire_data_change_event=False)
qgrid/grid.py:966: in _update_table
    double_precision=self.precision)
/home/erik/anaconda3/lib/python3.7/site-packages/pandas/io/json/_json.py:85: in to_json
    indent=indent,
/home/erik/anaconda3/lib/python3.7/site-packages/pandas/io/json/_json.py:291: in __init__
    self.schema = build_table_schema(obj, index=self.index)
/home/erik/anaconda3/lib/python3.7/site-packages/pandas/io/json/_table_schema.py:253: in build_table_schema
    fields.append(convert_pandas_type_to_json_field(s))
/home/erik/anaconda3/lib/python3.7/site-packages/pandas/io/json/_table_schema.py:117: in convert_pandas_type_to_json_field
    field["freq"] = arr.freqstr
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = index
2000-01-03    2000-01-03
2000-01-04    2000-01-04
2000-01-05    2000-01-05
2000-01-06    2000-01-06
2000-01-07  ...1-11    2000-01-11
2000-01-12    2000-01-12
2000-01-13    2000-01-13
2000-01-14    2000-01-14
Name: b, dtype: period[B]
name = 'freqstr'

    def __getattr__(self, name: str):
        """After regular attribute access, try looking up the name
        This allows simpler access to columns for interactive use.
        """
    
        # Note: obj.x will always call obj.__getattribute__('x') prior to
        # calling obj.__getattr__('x').
    
        if (
            name in self._internal_names_set
            or name in self._metadata
            or name in self._accessors
        ):
            return object.__getattribute__(self, name)
        else:
            if self._info_axis._can_hold_identifiers_and_holds_name(name):
                return self[name]
>           return object.__getattribute__(self, name)
E           AttributeError: 'Series' object has no attribute 'freqstr'

/home/erik/anaconda3/lib/python3.7/site-packages/pandas/core/generic.py:5274: AttributeError
```

--- a/README.rst
+++ b/README.rst
@@ -216,7 +216,7 @@ to do this.
 
 #. If desired, install the labextension::
 
-    jupyter labextension install js/
+    jupyter labextension link js/
 
 #. Run the notebook as you normally would with the following command::
 

--- a/js/package.json
+++ b/js/package.json
@@ -1,6 +1,6 @@
 {
   "name": "qgrid2",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "description": "An Interactive Grid for Sorting and Filtering DataFrames in Jupyter Notebook",
   "author": "Quantopian Inc.",
   "main": "src/index.js",
@@ -16,27 +16,28 @@
   ],
   "scripts": {
     "clean": "rimraf dist/",
-    "prepublish": "webpack",
+    "prepare": "webpack",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "jshint": "^2.9.5",
-    "rimraf": "^2.6.1",
-    "webpack": "^3.5.5"
+    "css-loader": "^3.4.2",
+    "expose-loader": "^0.7.5",
+    "file-loader": "^6.0.0",
+    "jshint": "^2.11.0",
+    "json-loader": "^0.5.7",
+    "rimraf": "^3.0.2",
+    "style-loader": "^1.1.3",
+    "webpack": "^4.42.0",
+    "webpack-cli": "^3.3.11"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1 || ^2",
-    "@jupyter-widgets/controls": "^1.0.0 || ^1.5.1",
-    "css-loader": "^0.28.7",
-    "expose-loader": "^0.7.3",
-    "file-loader": "^0.11.2",
+    "@jupyter-widgets/base": "^1.1 || ^2 || ^3",
+    "@jupyter-widgets/controls": "^1 || ^2",
     "jquery": "^3.2.1",
-    "jquery-ui-dist": "1.12.1",
-    "json-loader": "^0.5.4",
-    "moment": "^2.18.1",
+    "jquery-ui-dist": "^1.12.1",
+    "moment": "^2.24.0",
     "slickgrid-qgrid": "0.0.5",
-    "style-loader": "^0.18.2",
-    "underscore": "^1.8.3"
+    "underscore": "^1.9.2"
   },
   "jshintConfig": {
     "esversion": 6

--- a/js/src/qgrid.widget.js
+++ b/js/src/qgrid.widget.js
@@ -36,8 +36,8 @@ class QgridModel extends widgets.DOMWidgetModel {
       _view_name : 'QgridView',
       _model_module : 'qgrid',
       _view_module : 'qgrid',
-      _model_module_version : '^1.1.1',
-      _view_module_version : '^1.1.1',
+      _model_module_version : '^1.1.2',
+      _view_module_version : '^1.1.2',
       _df_json: '',
       _columns: {}
     });

--- a/js/webpack.config.js
+++ b/js/webpack.config.js
@@ -42,7 +42,8 @@ module.exports = [
             path: path.resolve(__dirname, '..', 'qgrid', 'static'),
             libraryTarget: 'amd'
         },
-        plugins: plugins
+        plugins: plugins,
+        mode: 'production'
     },
     {// Bundle for the notebook containing the custom widget views and models
      //
@@ -61,7 +62,8 @@ module.exports = [
             rules: rules
         },
         externals: ['@jupyter-widgets/base', '@jupyter-widgets/controls', 'base/js/dialog'],
-        plugins: plugins
+        plugins: plugins,
+        mode: 'production'
     },
     {// Embeddable qgrid bundle
      //
@@ -89,6 +91,7 @@ module.exports = [
             rules: rules
         },
         externals: ['@jupyter-widgets/base', '@jupyter-widgets/controls'],
-        plugins: plugins
+        plugins: plugins,
+        mode: 'production'
     }
 ];

--- a/qgrid/_version.py
+++ b/qgrid/_version.py
@@ -1,4 +1,4 @@
-version_info = (1, 3, 0, "final")
+version_info = (1, 3, 1, "final")
 
 _specifier_ = {"alpha": "a", "beta": "b", "candidate": "rc", "final": ""}
 

--- a/qgrid/grid.py
+++ b/qgrid/grid.py
@@ -566,8 +566,8 @@ class QgridWidget(widgets.DOMWidget):
     _model_name = Unicode('QgridModel').tag(sync=True)
     _view_module = Unicode('qgrid').tag(sync=True)
     _model_module = Unicode('qgrid').tag(sync=True)
-    _view_module_version = Unicode('1.1.1').tag(sync=True)
-    _model_module_version = Unicode('1.1.1').tag(sync=True)
+    _view_module_version = Unicode('^1.1.3').tag(sync=True)
+    _model_module_version = Unicode('^1.1.3').tag(sync=True)
 
     _df = Instance(pd.DataFrame)
     _df_json = Unicode('', sync=True)

